### PR TITLE
[STORELOCAT-25] Holiday exception hours updated

### PR DESCRIPTION
**Pull request by albertm805**
#### What problem is this solving?
- The store locator currently doesn't show any exception of hours or if is closed
- This was added in a separate section bellow the business hours

#### How to test it?
- You can test by visiting [this](https://beto--productusqa.myvtex.com/store/mexico-pickup-point-ciudad-de-mexico-03580/Mexico1) workspace

#### Screenshots or example usage:
Image 2022-07-01 at 12 21 26 p m

#### Image description
- The image shows only one holiday but there are 3 exception hours/holidays added
- One is that there are past holiday exceptions
- Another is that there is a valida holiday but without open and closing time (This will be set to closed)
- If there are business hours in this holiday then the hours will be set.

### CODE REVIEW

**Comment by polishq** [comment]
Instead of `Friday, 1`, can it say `Friday, July 1` (or whatever the month is)?

File: react/queries/getStore.gql [hunk 1]
```diff
@@ -5,6 +5,11 @@ query getStore($id: String!) {
       dayOfWeek
       closingTime
     }
+    pickupHolidays {
+      date
+      hourBegin
+      hourEnd
+    }
```
Comment (Arturo777) [inline react/queries/getStore.gql:12]: vtex.store-graphql already have this fields? if not, this app will break the store details

**Review by Arturo777 — CHANGES_REQUESTED** [review]

File: react/StoreHours.tsx [hunk 2]
```diff
@@ -94,6 +99,48 @@ const StoreHours: FC<WrappedComponentProps & StoreHoursProps> = ({
     return `${open} - ${close}`
   }
 
+  const displayHolidayDay = (item) => {
+    const holiday = new Date(item.date)
+    const dayOfWeek = holiday.getDay()
+    const dayOfMonth = holiday.getDate()
```
[the remark this answers is not in the document]
Comment (albertm805) [reply react/StoreHours.tsx:105]: Already did that.

File: react/queries/getStore.gql (hunk 1 above)
Comment (Arturo777) [inline react/queries/getStore.gql:12]: vtex.store-graphql already have this fields in the getStore query? if not, this app will break the store details, maybe we have to wait for it before publish this version.

**Review by Arturo777 — CHANGES_REQUESTED** [review]

**Review by Arturo777 — APPROVED** [review]
LGTM

**Review by polishq — APPROVED** [review]
LGTM

**Pull request merged**